src/Symfony/Bridge/Doctrine/ManagerRegistry.php

```diff
@@ -37,6 +39,35 @@ protected function getService($name)
      */
     protected function resetService($name)
     {
-        $this->container->set($name, null);
+        if (!$this->container->initialized($name)) {
+            return;
+        }
+        $manager = $this->container->get($name);
+
+        if (!$manager instanceof LazyLoadingInterface) {
+            @trigger_error(sprintf('Resetting a non-lazy manager service is deprecated since Symfony 3.2 and won\'t be supported in version 4.0. Set the "%s" service as lazy instead.', $name));
```

Comment:
The service may be marked as lazy but ProxyManager not installed, resulting in lazy services being non-lazy